runtime/executor/src/test/java/org/apache/nemo/runtime/executor/task/TaskExecutorTest.java

```diff
@@ -188,6 +189,24 @@ public void close() throws IOException {
     assertTrue(checkEqualElements(elements, runtimeEdgeToOutputData.get(taskOutEdge.getId())));
   }
 
+    /**
+   * Test invalid parameter failure.
+   */
+  @Test()
+  public void testInvalidInputData() throws Exception {
+    try{
+      // Execute the task.
+      final TaskExecutor taskExecutor = getTaskExecutor(null, null);
+      taskExecutor.execute();
+
+      // Check the output.
```

Comment:
Maybe change to:
// This should not be reached

Since we expect `taskExecutor.execute()` to throw `NullPointerException`?